conda/gateways/disk/link.py

```diff
@@ -74,7 +74,7 @@ def win_soft_link(src, dst):
     symlink = win_soft_link
 
 
-if not (on_win and PY2):
+if not (on_win and (PY2 or PYPY)):
```

Comment:
would it be better to check for `os.readlink` so that if PyPy ever implements it no change will be needed here?

Reply:
It's better for this check to be specific to the implementation than checking for `os.readlink`.

Since the alternative would be a potential surprise change of behavior if PyPy releases a new version and a user would upgrade to it. An implementation check allows better expectation management for Conda users and explicitly supported versions for the Conda project.